service/iot-service-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/service/FeedbackReceiverTest.java

```diff
@@ -36,7 +37,7 @@ public void constructor_input_hostName_null() throws Exception
         String deviceId = "xxx";
         IotHubServiceClientProtocol iotHubServiceClientProtocol = IotHubServiceClientProtocol.AMQPS;
         // Act
-        FeedbackReceiver feedbackReceiver = new FeedbackReceiver(hostName, userName, sasToken, iotHubServiceClientProtocol, deviceId);
+        FeedbackReceiver feedbackReceiver = new FeedbackReceiver(hostName, userName, sasToken, iotHubServiceClientProtocol, deviceId, null);
```

Comment:
If you have to pass null as an additional parameter, is that a breaking change?

Reply:
Looks like it, yeah. I've added an overloaded constructor so that it won't break now